lib/EpdFont/scripts/build-sd-fonts.py

```
@@ -0,0 +1,318 @@
+            if success:
+                # Count output files
+                family_dir = output_base / name
+                count = len(list(family_dir.glob("*.cpfont")))
+                size = sum(f.stat().st_size for f in family_dir.glob("*.cpfont"))
+                print(f"  OK: {name} ({count} files, {size / 1024 / 1024:.1f} MB)")
+            else:
+                print(f"  FAILED: {name}: {message}", file=sys.stderr)
+                failed.append(name)
+
+    # Summary
+    print(f"\n=== Summary ===\n")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)